server/aws-lsp-codewhisperer/src/language-server/codeWhispererServer.ts

```diff
@@ -324,6 +337,23 @@ export const CodewhispererServerFactory =
 
         const codePercentageTracker = new CodePercentageTracker(telemetry, telemetryService)
 
+        const codeDiffTracker: CodeDiffTracker<AcceptedInlineSuggestionEntry> = new CodeDiffTracker(
+            workspace,
+            logging,
+            (entry: AcceptedInlineSuggestionEntry, percentage, unmodifiedAcceptedCharacterCount) => {
+                telemetryService.emitUserModificationEvent({
+                    sessionId: entry.sessionId,
+                    requestId: entry.requestId,
+                    languageId: entry.languageId,
+                    customizationArn: entry.customizationArn,
+                    timestamp: new Date(),
```

Comment:
nit: for other events where we need to emit timestamp, we are calculating the same in TelemetryService. We can follow the same for this event?

Reply:
Also, do you mean to add date.now() here?

Reply:
I used `new Date()`, as it is in line with how it's done in VSCode Toolkit